pysal/network/util.py

```diff
@@ -109,6 +109,65 @@ def dijkstra(ntw, cost, node, n=float('inf')):
     return distance, np.array(pred, dtype=np.int)
 
 
+def dijkstra_mp((ntw, cost, node)):
+    """
+    Multiprocessing dijkstra()
+    Compute the shortest path between a start node and all other nodes in the web. 
+    This performs identically to `dijkstra()`, but utilizes multiple cores upon request.
+    
+    Parameters
+    ----------
+    ntw:        object
+                PySAL network object
+                
+    cost:       dict
+                key:    tuple
+                        (start node, end node)
+                value:  float
+                        Cost per edge to travel, e.g. distance
+    
+    node:       int
+                Start node ID
+    
+    n:          float('inf')
+                integer break point to stop iteration and return n neighbors
+    
+    Returns
+    -------
+    distance:   list
+                List of distances from node to all other nodes.
+    
+    pred:       list
+                List of preceeding nodes for traversal route.
+    """
+    
+    v0 = node
```

Comment:
It looks like this is identical to `dijkstra`, except for the arity of the arguments. Can we get away with:

```python
def dijkstra_mp((ntw, cost, node)):
    return dijkstra(ntw, cost, node)
```
to avoid having the same code in both places?